microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
+
+### UA-CH HTTPS Header
+
+By default, Chromium browsers like Edge will send the `Accept-CH-UA` request header in the following format: 
+
+```HTTP
+Sec-CH-UA: “Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”Dummy;Browser Brand”;v=”99”
+Sec-CH-UA-Mobile: ?0
+```
+Note: Client Hints are only sent over `HTTPS`. 
+
+More information, if needed, can be accessed by sending subsequent requests with the desired headers: 
+
+#### User-Agent Request & Response Headers
+| Response Header | Example Response Value |
+| ----------- | ----------- |
+|`Sec-CH-UA`|`“Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”GREASE”;v=”99"`|
+|`Sec-CH-UA-Mobile`|`false`|
+|`Sec-CH-UA-Full-Version`|`91.0.866.0`|
+|`Sec-CH-UA-Platform`|`Windows`|
+|`Sec-CH-UA Platform-Version`|`10.0`|
+|`Sec-CH-UA-Arch`|`x86`|
+|`Sec-CH-UA-Model`||
+
+### UA-CH JavaScript API
+
+The default return value from `navigator.userAgentData` is in the following format: 
+
+```javascript
+{brands: 
+    [{brand: “Chromium”,”version”:”91”}, 
+    brand: “Microsoft Edge”,”version”:”91”}, 
+    brand: “GREASE”,”version”:”99”},]
+mobile: false
+}
+```
+More detailed information, if needed, can be accessed with [getHighEntropyValues()](https://wicg.github.io/ua-client-hints/#getHighEntropyValues), for example, calling
+
+```javascript
+navigator.userAgentData.getHighEntropyValues(
+    ["architecture", "model", "platform", "platformVersion", "uaFullVersion"])
+    .then(ua => { console.log(ua) });
+```
+
+May output the following: 
+
+```javascript
+{architecture: "x86", 
+model: "", 
+platform: "Windows", 
+platformVersion: "10.0", 
+uaFullVersion: "92.0.866.0"}
+```
+
+## Best Practices for Using User Agent Client Hints
+Since the order brands appear in may change over time, developers are discouraged from hard-coding static checks for values in specific positions. To further prevent abuse of the brands list, GREASE values may also be sent, consisting of `null`, no value, or random values.
+
+If feature detection cannot be used, we recommend developers check for the Chromium brand as follows: 
+```javascript
+function isChromium() {
+    for (brand_version_pair in navigator.userAgentData.brands) {
+ 	    if (brand_version_pair.brand == "Chromium") {
+            return true;
+        }
+    }
+    return false;
+}
+```
+
+
+
+-------
+
+
+## User Agent String
+
+Whenever possible, Microsoft recommends developers to refactor any existing browser detection based on the User Agent string to User Agent Client Hints instead. However, for legacy reference, the format of the User Agent string is as follows:
+
 The `User-Agent` request header is in the following format:
 
 ```http
-User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.74 Safari/537.36 Edg/79.0.309.43
+User-Agent: Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Mobile Safari/537.36 Edg/90.0.818.46

```

Comment:
Given desktop Windows is where we have the most usage, I would not recommend replacing that wholesale with an Android mobile string. Can we instead additively call them out, e.g. "Windows desktop" and "Android phone"?

Reply:
Updated.